src/Messages/IPointToInterlockingConnection/IPointToInterlockingConnection.cs

```diff
@@ -0,0 +1,33 @@
+using System.Threading.Tasks;
+
+namespace EulynxLive.Messages.IPointToInterlockingConnection;
```

Comment:
we should keep the EulynxLive.Messages NuGet package for generated message definitions only (no behavior).